spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/partials/ErrorHandlingCommon.scala

```diff
@@ -54,4 +57,8 @@ trait ErrorHandlingCommon extends ErrorHandling {
     new Column(CaseWhen(branches))
   }
 
+  def errorColumnType(errorColumn: ErrorColumn, dataFrame: DataFrame): DataType = {
```

Comment:
That is a new function, does not implement the one declared in the parent trait (different signature).